layouts/partials/android-download.html

```diff
@@ -12,7 +12,6 @@
       <h3>{{ i18n "android_download_title" }}</h3>
     </div>
     <div class="px-4 py-8" x-data="{showShaSignature: false}">
-      <img class="inline-block mb-4" src="/img/android/app-icon.png" srcset="/img/android/app-icon.png 1x, /img/android/app-icon@2x.png 2x"/>
```

Comment:
Check if this asset is used anywhere else. If not, delete both `app-icon.png` and `app-icon@2x.png`.